integrations/plugins/plugin-gamma/plugin-gamma.php

```diff
@@ -40,6 +42,9 @@
 <div class="plugin-gamma-pre-content" style="border: 1px solid grey;">
   <h2>Plugin Gamma</h2>
   <p>Expected by plugin-gamma (v4.7.0 icon name): "fa fa-bathtub": <i class="fa fa-bathtub"></i></p>
+  <p class="group-icon">Also expected: before this paragraph, pseudo-element (v4.7.0 icon name group should become v5 users)</p>
+  <p class="facebook-icon">Also expected: before this paragraph, pseudo-element (v4.7.0 icon name facebook should become v5 facebook-f)</p>
+  <p class="hand-scissors-icon">Also expected: before this paragraph, pseudo-element (v4.7.0 icon name hand-scissors-o should become v5 hand-scissors)</p>
```

Comment:
And this v4.7 icon is supposed to correspond to to the `far` version in v5, not `fas`. Is there any way we can map it over to the correct style?